dune/pdelab/test/CMakeLists.txt

```diff
@@ -35,3 +35,9 @@ if(GMP_FOUND)
   pdelab_add_test(NAME testopbfem)
   pdelab_add_test(NAME test-dg-amg)
 endif(GMP_FOUND)
+
+# If we have MPI, include periodic tests
+if(MPI_FOUND)
+  pdelab_add_test(NAME testpoisson-periodic-2d COMMAND ${MPIEXEC} ${MPIEXEC_NUMPROC_FLAG} "4" testpoisson-periodic-2d)
```

Comment:
You can use `pdelab_add_test(NAME testpoisson-periodic-2d MPIRANKS 4)` for this - the CMake macro will take care of the MPI dependency. That way, you can also remove the `MPI_FOUND` test